WordPress/Classes/ViewRelated/People/InvitePersonViewController.swift

```diff
@@ -172,7 +172,9 @@ class InvitePersonViewController : UITableViewController {
             return
         }
 
-        roleViewController.mode = .Static(roles: Role.inviteRoles)
+        let roles = (blog.siteVisibility == .Private) ? Role.inviteRolesForPrivateSite : Role.inviteRoles
```

Comment:
Actually `Role.inviteRolesForPrivateSite` and `inviteRoles` are both arrays of `[Role]`, we kinda need that indirection